source/2020-05-15-the-ember-times-issue-148.md

```diff
@@ -117,14 +122,18 @@ Check out the full video explaining the `{{on}}` modifier on [EmberMap](https://
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Learn about JavaScript invoked helpers with the invokeHelper RFC ✨](https://github.com/emberjs/rfcs/pull/626)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Never seen a helper spawn from a JavaScript file before? Then you should check out [the latest RFC about the `invokeHelper` API](https://github.com/emberjs/rfcs/pull/626)!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The proposal suggests the creation of a new API, that allows you to invoke [helper functions](https://guides.emberjs.com/release/components/helper-functions/) via JavaScript and not only via templates (the only way how helpers can invoked in Ember apps today).
+This would extend Ember's reactive model by an API that
+
+* allows a transparent reuse of common code and patterns
+* is, similarly to components, owning a lifecycle
+* is, unlike components, JavaScript class _and_ template compatible
```

Comment:
```suggestion
This would extend Ember's reactive model by an API that:

* allows a transparent reuse of common code and patterns
* as, similarly to components, owning a lifecycle
* is, unlike components, JavaScript class _and_ template compatible.
```

not really sure of the correct punctuation, feel free to ignore